src/main/java/com/accelad/docx4j/tags/TagUpdaterTable.java

```diff
@@ -40,11 +42,11 @@ public boolean update(Tag tag) {
         int rowCount = valueTable.getRowCount();
         for (int i = 0; i < rowCount; i++) {
             TableRow row = new TableRow();
-            String[] rowValues = value[i];
+            TagValueTableRow rowValues = value.get(i);
 
-            for (String cellValue : rowValues) {
+            for (TagValueTableCell cellValue : rowValues) {
                 row.addCell(TableCell.builder()
-                        .withText(cellValue)
+                        .withText(cellValue.toString())
```

Comment:
it will be better if TableCell.Builder could build the TableCell from a TagValueTableCell, and not from a string value.